src/register.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { addons, types } from '@storybook/addons';
+import { AddonPanel } from '@storybook/components';
+
+import { ApolloClientPanel } from './addonPanel';
+import config from './config';
+
+const { parameterName, title } = config;
+const PANEL_ID = `${parameterName}/panel`;
+
+addons.register(parameterName, () => {
+  addons.add(PANEL_ID, {
+    title,
+    type: types.PANEL,
+    render: ({ active = false, key }) => (
```

Comment:
@brendanrygus I think we might want to follow the same convention the storybook team uses for panels here.

https://github.com/storybookjs/storybook/blob/master/addons/knobs/src/register.tsx

see how they pass in the API as a prop? and rather than rendering a component tree, it's a single component if we need to change anything in the future, this kind of pattern will give us more flexibility.

Reply:
Sure thing, makes sense to me

Reply:
This is updated now, forgot to comment last time 👍